packages/devtools_app/lib/src/screens/memory/panes/leaks/instrumentation/model.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:collection/collection.dart';
+
+enum LeakType {
+  notDisposed,
+  notGCed,
+  gcedLate,
```

Comment:
```suggestion
  notGCed,
  gcedLate,
```
```suggestion
  notGarbageCollected,
  garbageCollectedLate,
```
avoiding acronyms may help with readability in the future

Reply:
'GC' is the important concept of this feature (like html or http) that will be described in the central document. I prefer to leave the abbreviation in order to have shorter names. I added comments though. Thanks.